src/Microsoft.ML.Data/Transforms/NormalizerStaticExtensions.cs

```diff
@@ -0,0 +1,366 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Data.StaticPipe.Runtime;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace Microsoft.ML.Runtime.Data
+{
+    /// <summary>
+    /// Extension methods for static pipelines for normalization of data.
+    /// </summary>
+    public static class NormalizerStaticExtensions
+    {
+        private const long MaxTrain = Normalizer.Defaults.MaxTrainingExamples;
+        private const bool FZ = Normalizer.Defaults.FixZero;
+
+        /// <summary>
+        /// Learns an affine function based on the minimum and maximum, so that all values between the minimum and
+        /// maximum observed during fitting fall into the range of -1 to 1. Note that if values are later transformed
+        /// that are lower than the minimum, or higher than the maximum, observed during fitting, that the output
+        /// values may be outside the range of -1 to 1.
```

Comment:
this got me thinking. Maybe this should go into the extended help, but not intellisense help?

Reply:
Hmm OK I'll push that into remarks.